provider/equinix/environ.go

```diff
@@ -617,12 +619,23 @@ nextPlan:
 
 func validPlan(plan packngo.Plan, region string) bool {
```

Comment:
Can you add a small test to the suite for this change?

Reply:
@displague Any progress on this?

Reply:
@achilleasa how do I regerate the mocks? I used `make rebuild-schema` but this changed more of the generated content then it should have (just packngo updates).

Reply:
@displague Just run `go generate` in the equinix folder and you should be good to go.

Reply:
I have a failing test in the latest push. I'm not quite sure what I missed but it appears the expects for the API calls are not matching up.

Reply:
Let me grab that branch and take a look.